src/cljs/main/broadfcui/page/workspace/summary/catalog/questions.cljs

```diff
@@ -109,49 +110,61 @@
                            :rows 3
                            :data-test-id property})) ;; Dataset attribute, looks like "library:datasetOwner"
 
-(defn- render-ontology-typeahead [{:keys [prop colorize value-nullsafe set-property state property library-schema disabled]}]
-  (let [clear #(apply swap! state update :attributes dissoc property
-                      (library-utils/get-related-id+label-props library-schema property))]
+;; Needed to handle DS labels before adding the DS_URL field
+(defn- handle-differences [[related-id related-label]]
+  (if (nil? related-id)
```

Comment:
it's a stretch, but also check for nil on the label, to ensure they're always in sync?